specs/multispec.yml

```diff
@@ -8,6 +8,7 @@ specs:
       s2i_base: quay.io/centos7/s2i-core-centos7
       org: "centos7"
       prod: "centos7"
+      el_version: "7"
```

Comment:
Can we make do with `prod` instead? The Dockerfile changes use three branches (rhel, rhel8, centos7) so I do not see much reason to add a new attribute

Reply:
ok, I will use {% spec.prod == 'rhel7' or spec.prod == 'rhel8' %}